devops/vaults.env

```diff
@@ -20,6 +20,9 @@ VUE_APP_AUTH_API_URL="op://API/$APP_ENV/auth-api/AUTH_API_URL"
 VUE_APP_AUTH_API_VERSION="op://API/$APP_ENV/auth-api/AUTH_API_VERSION"
 VUE_APP_LEGAL_API_URL="op://API/$APP_ENV/legal-api/LEGAL_API_URL"
 VUE_APP_LEGAL_API_VERSION_2="op://API/$APP_ENV/legal-api/LEGAL_API_VERSION_2"
+VUE_APP_BUSINESS_API_GW_URL="op://API/$APP_ENV/business-api/BUSINESS_API_GW_URL"
+VUE_APP_BUSINESS_API_VERSION_2="op://API/$APP_ENV/business-api/BUSINESS_API_VERSION_2"
+VUE_APP_BUSINESS_API_KEY="op://API/$APP_ENV/business-api/BUSINESS_API_KEY"
```

Comment:
I also had to configure the key in the GCP triggers config (key substitution). I did this for:
- Filings UI
- Create UI
- Edit UI
- Business Dashboard UI
- BRD UI
- Namerequest UI 

I also had to change the cloudbuild script (in devops folder) to copy over this key. I was only able to do this for:
- Filings UI
- Create UI
- Edit UI

I don't know how this works for BD UI, BRD UI or NR UI... when those respective tickets are worked on, something may have to be updated.